src/pages/components/CertificateViewDetails.js

```diff
@@ -0,0 +1,215 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import CertificateTimeInterval from './CertificateTimeInterval'
+
+import BgIconDateSVG from '../../assets/images/icon-date.svg'
+import BgIconTimeSVG from '../../assets/images/icon-time.svg'
+
+export default function CertificateViewDetails({
+  size,
+  start,
+  end,
+  energy,
+  eco2,
+}) {
+  return (
+    <>
+      <PaddedWrapperDiv>
+        <ContainerFlexWrapDiv>
+          {start && end && (
+            <FlexLargeDiv>
+              <ContainerFlexNoWrapDiv>
+                <GrowingDiv>
+                  <HeadingDiv>Start timestamp of energy use</HeadingDiv>
+                  <WrapDiv>
+                    <HalfWrap>
+                      <IconWrap>
+                        <IconDate></IconDate>Date
+                      </IconWrap>
+                      <Text>{start.split('T')[0].split('-').join('/')}</Text>
+                    </HalfWrap>
+                    <HalfWrap>
+                      <IconWrap>
+                        <IconTime></IconTime>Time
+                      </IconWrap>
+                      <Text>{start.split('T')[1].slice(0, 5)}</Text>
+                    </HalfWrap>
+                  </WrapDiv>
+                </GrowingDiv>
+                <CertificateTimeInterval
+                  sTimestamp={start}
+                  eTimestamp={end}
+                  bgColor="white"
+                />
+                <GrowingDiv>
+                  <HeadingDiv>End timestamp of energy use</HeadingDiv>
+                  <WrapDiv>
+                    <HalfWrap>
+                      <IconWrap>
+                        <IconDate></IconDate>Date
+                      </IconWrap>
+                      <Text>{end.split('T')[0].split('-').join('/')}</Text>
+                    </HalfWrap>
+                    <HalfWrap>
+                      <IconWrap>
+                        <IconTime></IconTime>Time
+                      </IconWrap>
+                      <Text>{end.split('T')[1].split('.')[0]}</Text>
+                    </HalfWrap>
+                  </WrapDiv>
+                </GrowingDiv>
+              </ContainerFlexNoWrapDiv>
+            </FlexLargeDiv>
+          )}
+          {energy && (
+            <FlexDiv>
+              <ContainerFullWidthWrapDiv>
+                <HeadingDiv>Electric energy use</HeadingDiv>
+                <WrapShortDiv>{energy / 1000000} kWh</WrapShortDiv>
+              </ContainerFullWidthWrapDiv>
+            </FlexDiv>
+          )}
+          {size && (
```

Comment:
`&&`

Reply:
Same question as before, is this better?